src/market/market_utils.cairo

```diff
@@ -758,6 +758,12 @@ fn validate_market_token_balance(data_store: IDataStoreDispatcher, market: Marke
 fn validate_markets_token_balance(data_store: IDataStoreDispatcher, market: Span<Market>) { //TODO
 }
 
+/// Validate that the positions can be opened in the given market
+/// # Parameters
+/// * `data_store`: dispatcher for the data store
+/// * `market`: the market to check
+fn validate_position_market(data_store: IDataStoreDispatcher, market: Market) {} // TODO
```

Comment:
Adding todo's here so we can implement it once the helper functions are available